src/apps/homescreen/homescreen.py

```diff
@@ -1,15 +1,12 @@
 from trezor import config, res, ui
-from trezor.ui.swipe import Swipe, degrees
 
 from apps.common import storage
 
 
 async def homescreen():
-    while True:
-        await ui.backlight_slide(ui.BACKLIGHT_DIM)
-        display_homescreen()
-        await ui.backlight_slide(ui.BACKLIGHT_NORMAL)
-        await swipe_to_rotate()
+    await ui.backlight_slide(ui.BACKLIGHT_DIM)
+    display_homescreen()
+    await ui.backlight_slide(ui.BACKLIGHT_NORMAL)
```

Comment:
This is probably incorrect, because I've found out it breakes the ability to do screenshots (and probably other things?). However, I'm not sure how to fix this. Should it stay in the loop? With a blocking call at the end? But waiting for what?

Reply:
// cc @jpochyla